### DISCUSSION TITLE
channel dead but consumer still present

### DISCUSSION THREAD

**Discussion by LeoDemoniac** [rabbitmq/rabbitmq-server#2905] [Other]
### zombie consumer
rabbit-server info :  RabbitMQ 3.6.8, Erlang 20.2.3

on the server machine:
```bash
# have one consumer on this queue
rabbitmqctl list_consumers
process-performance	<rabbit@host220.2.25872.3734>	amq.ctag-5kPJ2kQq6NhyWI2_iQ1ekg	true	250	[]
process-performance	<rabbit@host220.3.17213.1032>	amq.ctag--r1c0pyJ1I5Ui7c4tWHdsg	true	250	[]
# no num 2 channel , this command we these messages
rabbitmqctl list_channels 
<rabbit@host220.3.17213.1032>	fake	1	0
```
Queue settings:
```bash
durable: true
Policy:ha-all
Node: host72
Slaves: host220
```
consumer tag `r1c0pyJ1I5Ui7c4tWHdsg` is our process, but that can not get any message.
consumer tag `5kPJ2kQq6NhyWI2_iQ1ekg` is the zombie consumer.
From the plugins: rabbitmq_management。Filter the queue `process-performance`  , we only found the `r1c0pyJ1I5Ui7c4tWHdsg` consumer, no information about the `5kPJ2kQq6NhyWI2_iQ1ekg`.
And our queue had `156`  _`Unacked`_。

What can we do to resolve this problem?
Expect your reply!

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
RabbitMQ 3.6.8 is [four years old](https://www.rabbitmq.com/changelog.html). That series has been [out of support since May 2018](https://www.rabbitmq.com/versions.html). Our only recommendation is to [upgrade](https://www.rabbitmq.com/upgrade.html).

**Reply by LeoDemoniac** [reply]
Sorry disturbing you again. Are there  any methods we can do to close or kill the consumer without the channel?
